cli/flox-watchdog/src/main.rs

```diff
@@ -139,7 +140,6 @@ fn run(args: Cli) -> Result<(), Error> {
             },
         }
     }?;
-    debug!("registered termination interest");
```

Comment:
Removed this because we no longer have a "subscribe" type mechanism. The polling doesn't happen until we call `wait_for_termination` further down.